vllm/model_executor/models/internlm2.py

```diff
@@ -433,3 +435,59 @@ def load_weights(self, weights: Iterable[Tuple[str,
                 weight_loader(param, loaded_weight)
             loaded_params.add(name)
         return loaded_params
+
+
+class InternLM2ForRewardModel(InternLM2ForCausalLM):
+
+    def __init__(
+        self,
+        *,
+        vllm_config: VllmConfig,
+        prefix: str = "",
+        model_type=InternLM2Model,
```

Comment:
```suggestion
        model_type: Type[InternLM2Model] = InternLM2Model,
```

Missing type annotation